apis/kubedb/v1alpha2/druid_helpers.go

```diff
@@ -419,6 +458,10 @@ func (d *Druid) SetDefaults() {
 				d.setDefaultContainerResourceLimits(&d.Spec.Topology.Historicals.PodTemplate, DruidNodeRoleHistoricals)
 			}
 		}
+
+		if d.Spec.Topology.Brokers == nil {
+			d.Spec.Topology.Brokers = &DruidNode{}
+		}
```

Comment:
No storage needed for nodes other than historicals & middlemanagers ?

Reply:
No, only for data node i.e. historicals and middleManagers